docs/DeveloperGuide.md

```diff
@@ -239,6 +240,47 @@ _{more aspects and alternatives to be added}_
 _{Explain here how the data archiving feature will be implemented}_
 
 
+### Listing Events
+
+The List Events feature allows users to enter `listEvents` and update the UI display for events, showing a list of all 
+events sorted in their current order inside `ModelManager`. In the future, the command will take in a parameter,
+which specifies the field that the events list can be permanently sorted by.
+
+### Implementation
```

Comment:
I think this should be a 4th level heading as how it is for the undo/redo feature

```suggestion
#### Implementation
```